### DISCUSSION TITLE
Chrestomathy / Corpus of texts

### DISCUSSION THREAD

**Discussion by lagleki** [lojban/casnu#9] [General]
Might be considered a part of [The CLL](https://github.com/lojban/casnu/discussions/6) project
- which license can we use? Public domain/CC 4.0 is a must?
- diverse texts: 
  - fairy tales, world legacy
  - programming, examples of programming languages transpiled to Lojban
  - logical problems solved in lojban
  - math problems solved in lojban
  - physics/chemistry articles in lojban
- can such texts be used for learning/teaching lojban?
- representation of the material?
  - parallel texts

**Comment by olpa** [comment]
I used the following corpus: https://github.com/mhagiwara/zmifanva/tree/master/docs. License is unknown.

**Comment by rlpowell** [comment]
Y'all know about https://corpus.lojban.org/ , yeah?

**Reply by olpa** [reply]
Didn't know that, thanks for the link. However, note that Lojban corpus is Lojban-only, and the collection from zmifanva is sentence-aligned between English and Lojban.
